docs/release-notes.rst

```diff
@@ -29,6 +29,13 @@ v0.1.27 (Not yet released)
     - Update Readme to give more information to someone who wants to work on
       the project.
 
+.. Pull requests 141
+*24 November 2017*
```

Comment:
I think we can just update the date and pull request list above, since none of this has been released yet. I only put that date above as a temporary thing to be replaced when we do the actual release.